core/src/main/kotlin/io/rover/campaigns/core/tracking/AutoScreenTracker.kt

```diff
@@ -48,3 +46,8 @@ internal class AutoScreenTracker : Application.ActivityLifecycleCallbacks {
     override fun onActivityStopped(activity: Activity?) {}
     override fun onActivityCreated(activity: Activity?, savedInstanceState: Bundle?) {}
 }
+
+interface TrackableContentScreen {
```

Comment:
I believe there is an `Interfaces.kt` that this can be moved into. That's the current convention, anyway.